lib/modules/showImages.js

```diff
@@ -1181,90 +1185,26 @@ addModule('showImages', (module, moduleID) => {
 		// Selfposts already don't mark it, so either don't bother or add marking for selfposts.
 	}
 
-	function generateVideoExpando(expandoButton, options) {
-		const mediaLink = expandoButton.imageLink;
-		const wrapperDiv = document.createElement('div');
-		wrapperDiv.className = 'madeVisible';
-		mediaLink.wrapperDiv = wrapperDiv;
-
-		const playerDiv = document.createElement('div');
-		playerDiv.className = 'res-player';
-
-		wrapperDiv.appendChild(playerDiv);
-
+	async function generateVideoExpando(expandoButton, options) {
+		const filledOptions = {
+			autoplay: module.options.autoplayVideo,
+			controls: true,
+			frameRate: 24,
+			loop: false,
+			muted: true,
+			playbackRate: 1,
+			reversable: false,
+			time: 0
```

Comment:
Object spread can be used here instead of `Object.assign`, although it might be a bit less obvious what's going on:
```js
const filledOptions = {
  /* ... */
  time: 0,
  ...options
};
```